src/build.sh

```diff
@@ -200,8 +222,8 @@ function load_build_config()
 
 function parse_build_options()
 {
-  local long_options='help,info,menu,doc,ccache,cpu-scaling:,warnings::,save-log-to:,llvm'
-  local short_options='h,i,n,d,c:,w::,s:'
+  local long_options='help,info,menu,doc,ccache,cpu-scaling:,warnings::,save-log-to:,llvm,clean'
+  local short_options='h,i,n,d,c:,w::,s:,cl'
```

Comment:
I think `c` makes more sense here. For the `--cpu-scaling`, we can use `S` instead of `c`.